IBAnimatable/AnimatableTableViewController.swift

```diff
@@ -0,0 +1,111 @@
+//
+//  AnimatableTableViewController.swift
```

Comment:
You can remove these two lines. Take a look at the other headers of the project, we are just keeping the author lines.